samples/stats/main.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/census-instrumentation/opencensus-go/stats"
+	"github.com/census-instrumentation/opencensus-go/tags"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	// ---------------------------------------------
+	// CREATING/REGISTERING KEYS AND VIEWS
+	// ---------------------------------------------
+
+	// Creates keys
```

Comment:
We always use proper sentences in Go comments like:

// Create keys.

Some of these comments are pretty arbitrary and the code is clear from the API name. I'd remove the redundant ones.

Reply:
Actually I find it helpful as it splits the code nicely in the multiple steps required to create, register, subscribe to the views and retrieve the collected data.
Yes, the API names are clear once you get used to them but I think it is easier (put less cognitive load) for the first time reader to follow by having simple comments than trying to read the whole code and the params of the various methods to figure out what is going on.